### ISSUE TITLE
Heroku Deployment:  Error 

### ISSUE THREAD

**Issue by JaganKaartik** [JaganKaartik/wheel#18]
```
       Compilation failed:
       Hash: 64c83204417bd77a626a
       Version: webpack 4.46.0
       Time: 104558ms
       Built at: 07/27/2021 6:44:18 AM
        4 assets
       Entrypoint application = css/application-1efb69c6.css js/application-e3b7af38345dd6cb21d5.js js/application-e3b7af38345dd6cb21d5.js.map
        [17] ./app/javascript/src/contexts/user.js 2.53 KiB {0} [built]
        [21] ./app/javascript/src/contexts/auth.js 2.7 KiB {0} [built]
        [28] ./app/javascript/src/apis/axios.js 1.99 KiB {0} [built]
        [31] ./app/javascript/src/constants/note.js 1.38 KiB {0} [built]
        [32] ./app/javascript/src/constants/formValidationSchemas.js 960 bytes {0} [built]
        [35] ./app/javascript/src/constants/contact.js 1.02 KiB {0} [built]
        [39] ./app/javascript/src/apis/authentication.js 426 bytes {0} [built]
        [40] ./app/javascript/src/constants/formInitialValues.js 244 bytes {0} [built]
        [71] ./app/javascript/src/common/logger.js 189 bytes {0} [built]
        [73] ./app/javascript/src/reducers/user.js 310 bytes {0} [built]
        [75] ./app/javascript/src/reducers/auth.js 671 bytes {0} [built]
        [77] ./app/javascript/src/apis/registrations.js 285 bytes {0} [built]
       [128] ./app/javascript/packs/application.js 794 bytes {0} [built]
       [129] ./app/javascript/stylesheets/application.scss 39 bytes {0} [built]
       [130] ./app/javascript/src sync ^\.\/.*$ 3.2 KiB {0} [built]
           + 459 hidden modules
       
       ERROR in ./app/assets/images/EmptyNotesList.svg
       Module build failed (from ./node_modules/file-loader/index.js):
       TypeError: Cannot read property 'fileLoader' of undefined
           at Object.module.exports (/tmp/build_19fe32bd/node_modules/file-loader/index.js:14:28)
        @ ./app/javascript/src/components/Dashboard/Contacts/index.jsx 1:2138-2193 1:4606-4625
        @ ./app/javascript/src sync ^\.\/.*$
        @ ./app/javascript/packs/application.js
       Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js??ref--6-1!node_modules/postcss-loader/src/index.js??ref--6-2!node_modules/sass-loader/dist/cjs.js??ref--6-3!app/javascript/stylesheets/application.scss:
           Entrypoint mini-css-extract-plugin = *
           [0] ./node_modules/css-loader/dist/cjs.js??ref--6-1!./node_modules/postcss-loader/src??ref--6-2!./node_modules/sass-loader/dist/cjs.js??ref--6-3!./app/javascript/stylesheets/application.scss 53.9 KiB {0} [built]
               + 1 hidden module
```

**Comment by karthiknmenon** [comment]
@JaganKaartik Can you install this specific neetoUI version `1.20.18` and try deploying the same?

**Comment by karthiknmenon** [comment]
@JaganKaartik Please update the deployed URL here once the review fixes in #19 are done.

**Comment by JaganKaartik** [comment]
I have the fixed the review suggestions in #19 with #20 but I'm not able to deploy the changes to Heroku.

I am getting some strange errors, 

```
Browserslist >3.0 config used in other tools.
remote:        error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
remote:        info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
remote:
remote: -----> Build failed
remote:
remote:  !     Outdated Yarn lockfile
remote:
remote:        Your application contains a Yarn lockfile (yarn.lock) which does not
remote:        match the dependencies in package.json. This can happen if you use npm
remote:        to install or update a dependency instead of Yarn.
remote:
remote:        Please run the following command in your application directory and check
remote:        in the new yarn.lock file:
remote:
remote:        $ yarn install
remote:        $ git add yarn.lock
remote:        $ git commit -m "Updated Yarn lockfile"
remote:        $ git push heroku main
remote:
remote:        https://help.heroku.com/TXYS53YJ
remote:
remote:  !     Push rejected, failed to compile Node.js app.
remote:
remote:  !     Push failed
remote: Verifying deploy...
remote:
remote: !	Push rejected to calm-reaches-32775.
remote:
```
Despite doing the steps mentioned I'm still getting this error.

**Comment by JaganKaartik** [comment]
I also tried using neetoUI version `1.20.18` but still got the same webpack error. 

However, `Wheel` is using neetoUI version `1.0.27` and I was able to deploy a fresh wheel instance so I switched to this version.

**Comment by karthiknmenon** [comment]
> I have the fixed the review suggestions in #19 with #20 but I'm not able to deploy the changes to Heroku.
> 
> I am getting some strange errors,
> 
> ```
> Browserslist >3.0 config used in other tools.
> remote:        error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
> remote:        info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
> remote:
> remote: -----> Build failed
> remote:
> remote:  !     Outdated Yarn lockfile
> remote:
> remote:        Your application contains a Yarn lockfile (yarn.lock) which does not
> remote:        match the dependencies in package.json. This can happen if you use npm
> remote:        to install or update a dependency instead of Yarn.
> remote:
> remote:        Please run the following command in your application directory and check
> remote:        in the new yarn.lock file:
> remote:
> remote:        $ yarn install
> remote:        $ git add yarn.lock
> remote:        $ git commit -m "Updated Yarn lockfile"
> remote:        $ git push heroku main
> remote:
> remote:        https://help.heroku.com/TXYS53YJ
> remote:
> remote:  !     Push rejected, failed to compile Node.js app.
> remote:
> remote:  !     Push failed
> remote: Verifying deploy...
> remote:
> remote: !	Push rejected to calm-reaches-32775.
> remote:
> ```
> 
> Despite doing the steps mentioned I'm still getting this error.

I am not really sure why your previous attempt at this issue did not fix the deployment. However, #21  solves it.

**Comment by JaganKaartik** [comment]
@karthiknmenon Deployed Link: https://wheel-jagankaartik-production.herokuapp.com
